Sources/Rendering/OpenGL/ImageMapper/index.js

```diff
@@ -570,7 +575,7 @@ function vtkOpenGLImageMapper(publicAPI, model) {
         ptsArray[9] = ext[1];
         ptsArray[10] = nSlice;
         ptsArray[11] = ext[5];
-      } else {
+      } else if (ijkMode === SlicingMode.K) {
```

Comment:
@martinken this seems to be the root of the tests failing: should this code run if we are not doing slicing? (`currentSlicingMode = SlicingMode.None`), and not just if we are doing K slicing? I thought this was only for what we used to call Z slicing.